From 6529d07e4dfdac48b34aabbbc5ace86570da6bc6 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 21 Aug 2015 11:45:55 -0700
Subject: [PATCH] cdev: turn some info messages to debug to reduce syslog
 spamming

cdev "Set" info messages should probably debug level messages to
void spamming the syslog so frequently.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/thd_cdev.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/thd_cdev.cpp b/src/thd_cdev.cpp
index 98b498f..109874e 100644
--- a/src/thd_cdev.cpp
+++ b/src/thd_cdev.cpp
@@ -96,7 +96,7 @@ int cthd_cdev::thd_cdev_exponential_controller(int set_point, int target_temp,
 		}
 	}
 
-	thd_log_info("Set : %d, %d, %d, %d, %d\n", set_point, temperature, index,
+	thd_log_debug("Set : %d, %d, %d, %d, %d\n", set_point, temperature, index,
 			get_curr_state(), max_state);
 
 	thd_log_debug("<<thd_cdev_set_state %d\n", state);
@@ -153,7 +153,7 @@ int cthd_cdev::thd_cdev_set_state(int set_point, int target_temp,
 		if (ret < get_min_state())
 			ret = get_min_state();
 		set_curr_state_raw(ret, zone_id);
-		thd_log_info("Set : %d, %d, %d, %d, %d\n", set_point, temperature,
+		thd_log_debug("Set : %d, %d, %d, %d, %d\n", set_point, temperature,
 				index, get_curr_state(), max_state);
 		ret = THD_SUCCESS;
 	} else {
-- 
2.5.0

